docs/events.rst

```diff
@@ -62,6 +62,33 @@ can join that pool is popped. This registration has it's ``waiting_list``, ``wai
 or the first pool that the user can join.
 
 
+Permissions
+-----------
+
+Events mostly uses the default permissions in LEGO, but has some custom permission handlers.
+
+Permissions based on event type
+*******************************
+For events we want to limit the different types of events a user can create or edit. So for
+events, one can add append the `event type` on the ``create`` and ``edit`` keyword permissions.
+F.ex: ``/sudo/admin/events/create/social/`` gives the user only access to create events with
+`event_type = social`.
+
+If a user has the ``/sudo/admin/events/edit/social/``, they have access to change an event object's type `to` ``social``.
```

Comment:
I don't think I see a reason for this one. If you can create one event with type x and y, you should also be able to change events from x to y, and from y to x.

To me a `/sudo/admin/events/edit/social/` permission looks like a permission to edit all events with type social.

Is there ever a reason where `can edit type` and `can create type` is not the same?

Reply:
If there is a good reason to split them, then it is ok, but I just can't see any reasons.. :thinking:

Reply:
I see your point and I agree with the fact that changing from x to y and reverse should be possible. As of now, the user will need object permission to edit the event, and in addition they will need permission to create events with type y. 

I very much agree that this is weird, and I too would think /edit/social/ means editing social events. But in reality it means the user har permission to do an edit action with social type events.

I think checking create permissions make sense from a use perspective, but the reason it is like this is that it makes more sense since we find the required keyword permission based on the action (in this case `edit`). 

We could modify this to check create permissions for edit actions as well (which would remove the /edit/event_type/ permissions completely). I'm not really sure what makes more sense here though so if the consensus is the latter, I'm fine with changing it.

Reply:
Yeah, I see your point, and I definitely agree about the fact that it makes the code shorter and more generic.

My only concern is that there is _never_ a reason to give a user only one of the perms, and therefore it imo. is easier to just merge them.

The current impl works, so its fine. I just think it can save some possible confusion later.

Reply:
We probably only want to show types they are allowed to use as well, and then having a single perm per type makes it easier to do that in a "generic" way.

Reply:
But big +1 for docs :raised_hands: